definitions/consented/json/CaseEventToFields/CaseEventToFields.json

```diff
@@ -1321,6 +1320,7 @@
     "PageID": 1,
     "PageDisplayOrder": 1,
     "PageColumnNumber": 1,
+    "FieldShowCondition": "applicantRepresented=\"DONTSHOW\"",
```

Comment:
This info we always shows before start. Might need to revert

Reply:
check what? this is to be displayed